src/moisture_reading.py

```diff
@@ -0,0 +1,28 @@
+import eel
```

Comment:
Why is eel being imported here when we don't currently use it?

Reply:
We will use it shortly! Could have left it out for now